src/main/java/Duke.java

```diff
@@ -1,10 +1,94 @@
+import java.io.*;
```

Comment:
Imported classes should always be listed explicitly according to Java coding standard.